Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace MathFormula
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Please, write x: ");
+            double x = Convert.ToDouble(Console.ReadLine());
+
+            double sin = Math.Sin(Math.Sqrt(Math.Pow(x, 2) + 4) / 2);
+            double cos = Math.Cos(Math.Sqrt(Math.Pow(x, 2) + 1) / 2); 

```

Comment:
Квадрат числа проще вычислять через умножение
```suggestion
            double sin = Math.Sin(Math.Sqrt(x * x + 4) / 2);
            double cos = Math.Cos(Math.Sqrt(x * x + 1) / 2); 

```